### PR TITLE
Update Information

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
**This pull request about:**

- Update HTML file.
- Update CSS file.
- Added Project materials.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,28 +1,187 @@
...
+          <h3 class="projecttitle">Projects name goes here</h3>
+          <p class="languagebadge">
+            <span class="language">HTMLS/CSS</span>
+            <span class="language">Ruby on Rails</span>
+            <span class="language">JavaScript</span>
+          </p>
+          <br />
+          <button type="button" class="btn1">See this project</button>
+        </div>
+      </div>
```
Comment (Cathella) [inline index.html:52]: - [x] You have done a good job on this project so far! Kudos to you 🔥. Per project requirements, the project contents are to be aligned with `flexbox`. I request that you implement it 😄.

File: index.html [hunk 2]
```diff
@@ -1,28 +1,187 @@
...
+    </section>
+    <section class="about" id="about">
+      <h2 class="aboutheader">About</h2>
+      <p class="introabout">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
+        eiusmod
```
Comment (Cathella) [inline index.html:145]: - [x] You did a great job on the about section! And I highly recommend that you continue to adjust your design to match the Figma design

| Your design | Figma design|
| --- | --- |
| abt | abt2 |
| - I suggest that you increase on the font size of the heading | ✔️  |
| - Also work on the overall alignment of the elements to match those on the right | ✔️  |

File: index.html [hunk 3]
```diff
@@ -1,28 +1,187 @@
...
+          <ul>
+            <li class="about-title">
+              <h3>languages</h3>
+            </li>
+            <li>JavaScript (es5/es6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
```
Comment (Cathella) [inline index.html]: - [x] Great work done on implementing the technology categories. However, we still have work to do, starting from aligning items to the left and giving them an extreme margin left to changing the font-family to match that of the Figma design.


| Your design | Figma design |
| --- | --- |
| lag | lag2 |
| Please implement the changes for the rest of the sections too | ✔️  |

**Review by Cathella — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED 🟡

### Hi @mukhammadrizooff  👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## The Good Points 👏🏻

✔️ No linter errors
✔️ Used correct GitHub flow
✔️ Used CSS grid to place elements in the page.
✔️ Used the `<section>` tag
✔️ The section uses the font and colors defined for the template

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mukhammadrizooff) [reply index.html:52]: I have already used it

File: index.html (hunk 2 above)
Comment (mukhammadrizooff) [reply index.html:145]: The said work has been done.

File: index.html (hunk 3 above)
Comment (mukhammadrizooff) [reply index.html]: This work has been done.

**Review by brytebee — APPROVED** [review]
Hi @mukhammadrizooff 👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**